platform/linux-generic/include/odp_ipsec_internal.h

```diff
@@ -84,8 +84,22 @@ int _odp_ipsec_status_send(odp_queue_t queue,
 
 #define IPSEC_MAX_SALT_LEN	4    /**< Maximum salt length in bytes */
 
-/* 32 is minimum required by the standard. We do not support more */
-#define IPSEC_ANTIREPLAY_WS	32
+/* The minimum supported AR window size, we do not support less */
+#define IPSEC_AR_WIN_SIZE_MIN	32
+
+/* The maximum supported AR window size, we do not support more */
```

Comment:
The "we do not support more" and "we do not support less" parts are redundant (these are already described and named as maximum and minimum) and could be removed.

Reply:
ack, will remove these comments.